mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -485,15 +494,17 @@ static llvm::cl::opt<float>
 static llvm::cl::opt<float>
     relDiffThreshold("relDiff_threshold",
                      llvm::cl::desc("Threshold for relDiff metric"),
-                     llvm::cl::value_desc("error"), llvm::cl::init(100.0f));
+                     llvm::cl::value_desc("error"), llvm::cl::init(0.000001f));
 static llvm::cl::opt<std::string> printVerifyResults(
```

Comment:
We could define an `enum class VerificationPrinting : char` and then `static_cast` it away instead of string switching?